openbas-api/src/main/java/io/openbas/importer/V1_DataImporter.java

```diff
@@ -1069,6 +1072,65 @@ private void importInjects(
     }
   }
 
+  public static ContractOutputType formatStringToContractOutputType(String value) {
+    for (ContractOutputType type : ContractOutputType.values()) {
```

Comment:
Is there a reason to create this function ? It looks very similar to what 
```suggestion
    ContractOutputType.valueOf(value);
``` would do